### PR TITLE
move atari wrapper to examples and publish v0.2.4

### PR DESCRIPTION
**Pull request by Trinkle23897**
Remove atari wrapper inside core. Move it to `examples/`. It is @Mehooz 's work.
I also change the seed in test_drqn. In our local test, test_drqn behaves normally as before but in GitHub Actions it seems quite unstable under previous seed.

### CODE REVIEW

File: examples/atari.py [hunk 1]
```diff
@@ -14,6 +18,30 @@ def create_atari_environment(name=None, sticky_actions=True,
     return env
 
 
+def preprocess_fn(obs=None, act=None, rew=None, done=None,
+                  obs_next=None, info=None, policy=None):
+    if obs_next is not None:
+        shape = np.shape(obs_next)
+        obs_next = np.reshape(
+            obs_next, (shape[0] * shape[1], shape[2], shape[3]))
```
Comment (duburcqa) [inline examples/atari.py]: Why not `np.reshape(obs_next, (-1, *obs_next.shape[2:]))` ? As you wish

File: examples/atari.py [hunk 2]
```diff
@@ -14,6 +18,30 @@ def create_atari_environment(name=None, sticky_actions=True,
...
+        obs_next = obs_next.transpose(1, 2, 0)
+        obs_next = cv2.resize(obs_next, (SIZE, SIZE))
+        obs_next = np.asarray(obs_next, dtype=np.uint8)
```
Comment (duburcqa) [inline examples/atari.py]: `asanyarray` is usually a better choice.

File: examples/atari.py [hunk 3]
```diff
@@ -14,6 +18,30 @@ def create_atari_environment(name=None, sticky_actions=True,
...
+        obs_next = np.reshape(obs_next, (-1, FRAME, SIZE, SIZE))
+        obs_next = obs_next.transpose(0, 2, 3, 1)
```
Comment (duburcqa) [inline examples/atari.py]: I would prefer `np.moveaxis` for clarity.

File: examples/atari.py [hunk 4]
```diff
@@ -14,6 +18,30 @@ def create_atari_environment(name=None, sticky_actions=True,
...
+    elif obs is not None:
+        shape = np.shape(obs)
+        obs = np.reshape(obs, (shape[0] * shape[1], shape[2], shape[3]))
+        obs = obs.transpose(1, 2, 0)
```
Comment (duburcqa) [inline examples/atari.py]: same, `np.moveaxis` is clearer in my opinion.

File: examples/atari.py [hunk 5]
```diff
@@ -14,6 +18,30 @@ def create_atari_environment(name=None, sticky_actions=True,
...
+        obs_next = np.reshape(
+            obs_next, (shape[0] * shape[1], shape[2], shape[3]))
+        obs_next = obs_next.transpose(1, 2, 0)
+        obs_next = cv2.resize(obs_next, (SIZE, SIZE))
+        obs_next = np.asarray(obs_next, dtype=np.uint8)
+        obs_next = np.reshape(obs_next, (-1, FRAME, SIZE, SIZE))
+        obs_next = obs_next.transpose(0, 2, 3, 1)
+    elif obs is not None:
+        shape = np.shape(obs)
+        obs = np.reshape(obs, (shape[0] * shape[1], shape[2], shape[3]))
```
Comment (duburcqa) [inline examples/atari.py]: same.

**Comment by Trinkle23897** [comment]
I think it is time to publish v0.2.4.

**Review by duburcqa — APPROVED** [review]

**Comment by duburcqa** [comment]
I guess you are right (maybe specify it in the PR message).

**Pull request merged**